### ISSUE TITLE
Warn/error on submit to an unreachable Ketrew server

### ISSUE THREAD

**Issue by ihodes** [hammerlab/ketrew#311]
Currently just hangs when submitting to e.g. incorrecpt http://ketrew.server instead of the correct https://ketrew.server.

**Comment by iskandr** [comment]
Just ran into the same problem. It seems like there should be an initial handshake with a short timeout (followed by an informative error). Instead ketrew just hangs indefinitely.

**Comment by smondet** [comment]
I'm not sure this is technically possible reliably; if try an HTTPS server with chrome or firefox as clients, they also take ages to fail (like I had far enough time to write this comment and it's still spinning…).
I guess we can set a much lower timeout; but then we may hit false positives.

**Comment by hammer** [comment]
@smondet what is this blocked on?

**Comment by smondet** [comment]
@hammer well, it seems to be the HTTP/TLS/TCP Vs HTTP/TCP stacks that work that way.

If I do `wget http://github.com:443/` (i.e. try to HTTP directly on a TLS server) on a I also get a very long time before failure.

**Comment by hammer** [comment]
@smondet can we put something on port 80 that redirects to 443?

**Comment by smondet** [comment]
@hammer the problem is when we already choose a port in the URL (like I did with `http://github.com:443/`). Also, when we use Ketrew we are often on non-standard ports anyway (since it's often practical to run as a regular user).

**Comment by hammer** [comment]
Okay fair enough. Let's close this as won't fix then.

**Comment by hammer** [comment]
Actually sorry let's add something to the Ketrew documentation then close?
